ios/RNPurchases.m

```diff
@@ -467,6 +467,27 @@ static void logUnavailablePresentCodeRedemptionSheet() {
     #endif
 }
 
+RCT_EXPORT_METHOD(showManageSubscriptions:
+                  (RCTPromiseResolveBlock)resolve
+                  reject:(RCTPromiseRejectBlock)reject) {
+    if (@available(iOS 13.0, macOS 10.15, visionOS 1.0, *)) {
```

Comment:
I think we will also need to include the `#if TARGET_OS_IPHONE && !TARGET_OS_TV` macros wrapping this (and return an error otherwise). This is because we have macros making it so these symbols won't be available in all platforms.